### DISCUSSION TITLE
Моё уважение!

### DISCUSSION THREAD

**Discussion by TalismanFR** [payuru/php-payu4#5] [General]
Мои комплименты всем причастным к либе и v4! После плоских массивов в form-data предыдущих версий с неструктурированым ответом, увидеть такую качественную реализаци очень приятно.

**Comment by payuru** [comment]
@TalismanFR Огромное спасибо за отзыв!
Приятно знать что проделанная работа пришлась по вкусу нашим партнерам.

@Doctorrr спасибо всей команде разработчиков!
